src/main/java/net/sf/jsqlparser/util/deparser/AlterViewDeParser.java

```diff
@@ -13,25 +13,25 @@
 import net.sf.jsqlparser.statement.select.PlainSelect;
 import net.sf.jsqlparser.statement.select.SelectVisitor;
 
-public class AlterViewDeParser {
+public class AlterViewDeParser extends AbstractDeParser<AlterView> {
 
-    protected StringBuilder buffer;
     private SelectVisitor selectVisitor;
 
     public AlterViewDeParser(StringBuilder buffer) {
+        super(buffer);
         SelectDeParser selectDeParser = new SelectDeParser();
```

Comment:
should we make SelectDeParser also take the buffer instead of calling `setBuffer` ?

Reply:
I don't care, but if it's for consistency, we could implement it...